### ISSUE TITLE
Spell Muncher not importing all spells

### ISSUE THREAD

**Issue by thegatherer13** [MrPrimate/ddb-importer#187]
Hi Jack,
Thanks for working on such a helpful module for us all!! 

The Spell Muncher seems to be missing a number of spells for me. I've refreshed the browser page and tried multiple times using various combinations of which picture to use or use none at all, but I have spells like Blade Ward, Friends, and Infestation missing whilst others from the same sources are there (such as Eldritch Blast and Thunderclap).

Most of the functionality seems to be working fine for me - the Item Muncher and Monster Muncher are generating really long lists, but I haven't checked everything item by item.

I've looked through the closed tickets here and the FAQ and couldn't see a fix for this, so apologies if I've missed something and an re-raising an addressed issue.

Foundry VTT version: 0.8.9
DDB Importer version: 2.6.23

**Comment by MrPrimate** [comment]
Is the content shared with you or do you own it on DDB? If yes, do you have the right campaign selected in the core settings?

**Comment by thegatherer13** [comment]
Hi Jack,

Thanks for the quick response - I'm thinking you're right that it must be down to what I do and don't own. I got thrown off as the DDB Muncher lists looked more comprehensive than the built-in Foundry VTT SRD ones. I did wonder if it was tied in with which source materials I'm missing on D&D Beyond, but couldn't see anyone else asking that question and I didn't have the spare $20 to buy one of my missing ones as a test.

Sorry for wasting your time, and thanks again for getting back to me to quickly to clear this up!
